pool/app/custom_field/entity.ml

```diff
@@ -535,6 +454,139 @@ module Public = struct
   ;;
 end
 
+module Group = struct
+  module Id = struct
+    include Pool_common.Id
+
+    let schema () =
+      Pool_common.Utils.schema_decoder
+        (Utils.fcn_ok of_string)
+        value
+        Message.Field.CustomFieldGroup
+    ;;
+  end
+
+  type t =
+    { id : Id.t
+    ; model : Model.t
+    ; name : Name.t
+    }
+  [@@deriving eq, show]
+
+  let create ?(id = Id.create ()) model name = { id; model; name }
+
+  let name lang (t : t) =
+    Name.find_opt lang t.name |> CCOption.get_exn_or "Cannot find field name."
```

Comment:
No this will not be passed to the frontend. it will throw an exception. using a result will not be possible as this function is normally called from the views. We need to implement a check whenever a new language is activated to make sure that all translations are present (also in the i18n module). The issue already exists.